Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1080,22 +1099,30 @@ def get_user_tokens(user_id):
 
 
 def search_all_mailboxes():
-    next_page_token = None
+    users_next_page_token = None
     service = get_service('admin', 'directory_v1')
+    receive_only_accounts = argToBoolean(demisto.args().get('show-only-mailboxes', 'false'))
```

Comment:
Please refrain from accessing `demisto.args()` outside of `main` - you can pass `show-only-mailboxes` arg.